assets/helpers/payPalExpressCheckout/payPalExpressCheckoutActions.js

```diff
@@ -42,24 +48,32 @@ function handleSetupResponse(response: Object) {
   return resp;
 }
 
-// First Step: setupPayment
-// Sends request to server to setup payment, and returns Paypal token.
-function setupPayment(dispatch, state: Object) {
+function payPalRequestData(bodyObj: Object, state: Object) {
+
+  const body = JSON.stringify(bodyObj);
+
+  return {
+    credentials: 'include',
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json', 'Csrf-Token': state.csrf.token },
```

Comment:
If the CSRF token is all that `state` is used for, could the argument be `csrfToken` instead, rather than passing the full state?